docs/OV_Runtime_UG/migration_ov_2_0/configure_devices.md

```diff
@@ -2,9 +2,9 @@
 
 ### Introduction
 
-Inference Engine API provides an [ability to configure devices](https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_InferenceEngine_QueryAPI.html) via configuration keys and [get device specific metrics](https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_InferenceEngine_QueryAPI.html#getmetric). The values taken from `InferenceEngine::Core::GetConfig` are requested by its string name, while return type is `InferenceEngine::Parameter` and users don't know what is the actual type is stored in this parameter.
+The Inference Engine API provides an ability to [configure devices](https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_InferenceEngine_QueryAPI.html) via configuration keys and [get device specific metrics](https://docs.openvino.ai/2021.4/openvino_docs_IE_DG_InferenceEngine_QueryAPI.html#getmetric). The values taken from `InferenceEngine::Core::GetConfig` are requested by their name strings, while the return type is `InferenceEngine::Parameter` making users lost on what the actual type stored in this parameter is.
```

Comment:
`their name strings` - how to read it?

Reply:
I don't know exactly what to do with it to make it unambiguous. I tried the plural version but it is as cryptic as before... What exactly do we want to say in this line?

Reply:
Changed to "are requested by the string name" which does not eliminate ambiguity but is a safer option.